lib/stripe/multipart_encoder.rb

```diff
@@ -35,10 +35,9 @@ def self.encode(params)
 
     # Initializes a new multipart encoder.
     def initialize
-      # We seed this with an empty string so that it encoding defaults to UTF-8
-      # instead of ASCII. The empty string is UTF-8 and new string inherits the
-      # encoding of the string it's seeded with.
-      @body = String.new("")
+      # Kind of weird, but required by Rubocop because the unary plus operator
+      # is considered faster than `Stripe.new`.
+      @body = +""
```

Comment:
This change is a little weird, but RuboCop insisted, so I just did it. Added comment above because seeing a unary `+` on a string is so unusual and is probably be confusing to the 99.9%.

Reply:
Indeed. TIL!